Recompile script does not run on Windows

Running `npm run recompile` fails with the following message:

```
> dt-stability@0.0.1 clean C:\Code\stability-js
> rm -rf .compiled && mkdir -p .compiled/src && mkdir -p .compiled/tests

The syntax of the command is incorrect.

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "clean"
npm ERR! node v6.10.3
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! dt-stability@0.0.1 clean: `rm -rf .compiled && mkdir -p .compiled/src && mkdir -p .compiled/tests`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the dt-stability@0.0.1 clean script 'rm -rf .compiled && mkdir -p .compiled/src && mkdir -p .compiled/tests'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the dt-stability package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rm -rf .compiled && mkdir -p .compiled/src && mkdir -p .compiled/tests
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs dt-stability
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls dt-stability
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Code\stability-js\npm-debug.log

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "recompile"
npm ERR! node v6.10.3
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! dt-stability@0.0.1 recompile: `npm run clean && npm run compile`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the dt-stability@0.0.1 recompile script 'npm run clean && npm run compile'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the dt-stability package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run clean && npm run compile
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs dt-stability
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls dt-stability
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Code\stability-js\npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
 		"lint": "eslint ./src ./tests",
 		"lintfix": "eslint --fix ./src ./tests",
 		"recompile": "npm run clean && npm run compile",
-		"clean": "rm -rf .compiled && mkdir -p .compiled/src && mkdir -p .compiled/tests",
+		"clean": "rm -rf .compiled && mkdir .compiled && cd .compiled && mkdir src && mkdir tests",
 		"compile": "babel src --out-dir .compiled/src && babel tests --out-dir .compiled/tests",
 		"coveralls": "cat coverage/lcov.info | coveralls",
 		"test": "cross-env NODE_ENV=test nyc --reporter=lcov mocha tests"

